src/net_processing.cpp

```diff
@@ -3598,11 +3539,12 @@ bool PeerLogicValidation::SendMessages(CNode* pto)
             }
         }
 
-        TRY_LOCK(cs_main, lockMain); // Acquire cs_main for IsInitialBlockDownload() and CNodeState()
```

Comment:
Why remove this comment?

Reply:
`IsInitialBlockDownload` does *not* require cs_main